specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2024-03-03/gallery.json

```diff
@@ -1737,1525 +1737,2518 @@
+      "get": {
+        "tags": [
+          "GalleryInVMAccessControlProfiles"
+        ],
+        "operationId": "GalleryInVMAccessControlProfiles_Get",
+        "description": "Retrieves information about a gallery inVMAccessControlProfile.",
+        "parameters": [
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "galleryName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Shared Image Gallery from which the InVMAccessControlProfiles are to be retrieved."
+          },
+          {
+            "name": "inVMAccessControlProfileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the gallery inVMAccessControlProfile to be retrieved."
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GalleryInVMAccessControlProfile"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common-types/v1/common.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a gallery inVMAccessControlProfile.": {
+            "$ref": "./examples/galleryResourceProfileExamples/GalleryInVMAccessControlProfile_Get.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "GalleryInVMAccessControlProfiles"
+        ],
+        "operationId": "GalleryInVMAccessControlProfiles_Delete",
+        "description": "Delete a gallery inVMAccessControlProfile.",
+        "parameters": [
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "galleryName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "he name of the Shared Image Gallery in which the InVMAccessControlProfile resides."
+          },
+          {
+            "name": "inVMAccessControlProfileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the gallery inVMAccessControlProfile to be deleted."
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
```

Comment:
Pl refer to https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#response-2    for delete response guidelines

Reply:
this delete is an LRO, and we have 202 and 204 as the response codes which is what the reference also suggests. am I missing something?

Reply:
sure, for always async delete this is fine, if you dont have a sync path.

Reply:
yes, this is async only